mod/account/set/right/password.rb

```diff
@@ -5,8 +5,9 @@
 PASSWORD_REGEX = {
   lower: /[a-z]/,
   upper: /[A-Z]/,
-  special_char: /[!@#$%^&*()]/,
-  number: /\d+/
+  special_char: /[!§$%&#]/,
```

Comment:
No objection, but interested to hear what inspired this change?

Reply:
Ah, I see. I see that she used the word "like".  I don't think she meant to imply that we would use _only_ those characters.  This would be a pretty weird, random set. I think she meant to give some examples even though we might support a lot more.

Is that third character non-ascii?  This discussion gives some decent reasons for not using unicode (https://security.stackexchange.com/questions/225346/is-it-bad-to-use-special-characters-in-passwords). And I certainly don't think we should use only one unicode character.

In this case I think perhaps we revert and discuss with Melanie.

Reply:
It might be good to add this link to pt so she can look at it before the discussion.